GliaWidgets/Sources/OperatorRequestHandlerService/OperatorRequestHandlerService+Mock.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+#if DEBUG
+extension OperatorRequestHandlerService {
+    static let mock: OperatorRequestHandlerService = .init(
```

Comment:
Having reference as static global instance can and will be a problem known as "bleeding unit test", where shared state affects other unit tests because it outlives the scope of unit test. I would recommend to rewrite it as static function with parameters. [Here's](https://github.com/salemove/ios-sdk-widgets/blob/a1566e8cac0a036c709ee1d9ecbd57f0e039c81a/GliaWidgets/Sources/Upload/FileUploader.Mock.swift#L3) an example of such function.

Reply:
fixed [2bff344](https://github.com/salemove/ios-sdk-widgets/pull/885/commits/2bff344a639d14c0048afecba3007635c01d515e)